benchmarks/benchmarks/optimize_zeros.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import division, print_function, absolute_import
 
-from math import sqrt
+from math import sqrt, exp, cos, sin
 
 # Import testing parameters
 try:
     from scipy.optimize._tstutils import methods, mstrings, functions, fstrings
 except ImportError:
     pass
+try:
+    from scipy.optimize.zeros import newton
+except ImportError:
+    pass
```

Comment:
I don't think this try/except block is needed since `newton` exists for a very long time ( 6e39a213abc09ab7e4c9823f037dfdaf42b2b900). Also, it can be imported from `scipy.optimize` directly.

Reply:
Hi @dlax, I was following the [asv benchmarks readme](../tree/master/benchmarks#writing-benchmarks), but I think I understand: because the asv benchmarks were added in commit 51127e0ff3d06c6c18d3d110516b13da05ee77a5 on 2015-02-07, many years _after_ `newton`, so then there aren't any benchmarks that wouldn't be able to import it. OK, I'll remove this try/except. Thanks again!